packages/@okta/vuepress-site/docs/guides/refresh-tokens/get-refresh-token/index.md

```diff
@@ -51,22 +51,30 @@ https://${yourOktaDomain}/oauth2/default/v1/authorize?client_id=${clientId}
 The following is an example request to the `/token` endpoint to obtain an access token, an ID token (by including the `openid` scope), and a refresh token for the [Authorization Code flow](/docs/guides/implement-grant-type/authcode/main/). The value for `code` is the authorization code that you receive in the response from the request to the `/authorize` endpoint.
 
 ```bash
-POST https://${yourOktaDomain}/oauth2/default/v1/token?grant_type=authorization_code
-&redirect_uri=yourApp%3A%2Fcallback
-&code=DPA9Utz2LkWlsronqehy
-&state=9606b31k-51d1-4dca-987c-346e3d8767n9
-&scope=openid%20offline_access
+curl --location --request POST 'https://${yourOktaDomain}/oauth2/default/v1/token' \
+--header 'Accept: application/json' \
+--header 'Authorization: Basic ${Base64(<client_id>:<client_secret>)}' \
+--header 'Content-Type: application/x-www-form-urlencoded' \
+--data-urlencode 'grant_type=authorization_code' \
```

Comment:
It should just use `-F`, the content-type of the request should be `application/x-www-form-urlencoded`
BTW, it seems other places in oauth2 guide are using `-d` which is incorrect, you might submit a JIRA to fix them or just do it in this PR.

Reply:
👍 we had a chat about `-d` is correct in this case